package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Sia-UI",
-  "version": "v1.0.2",
+  "version": "v1.0.0",
```

Comment:
Aren't we working on v1.0.2 of the ui? Is there any reason we are keeping the old version string, it could make user debugging more difficult?